src/base/static/client/mapseed-api-client.js

```diff
@@ -1,3 +1,64 @@
+const status = response => {
+  if (response.status >= 200 && response.status < 300) {
+    return Promise.resolve(response);
+  } else {
+    return Promise.reject(new Error(response.statusText));
+  }
+};
+
+const json = response => response.json();
+
+const getDatasets = async datasetUrls =>
+  Promise.all(datasetUrls.map(url => fetch(url).then(json)));
+
+const createPlaceTag = async (url, tagData) => {
+  try {
+    return await fetch(`${url}/tags`, {
+      headers: {
+        "Content-Type": "application/json",
+      },
+      method: "POST",
+      body: JSON.stringify(tagData),
+    })
+      .then(status)
+      .then(json);
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.log("Error: Failed to create tag.", err);
```

Comment:
These should be `console.error`

Reply:
Yeah, I was wondering about error handling in general. I'm not sure we need to crash though. I think if the request fails the view should just reflect the old state before the request was attempted. Some sort of messaging would be nice too.